packages/prosess-beregningsgrunnlag/src/components/ytelsesspesifikkseOpplysninger/frisinn/FrisinnUtils.jsx

```diff
@@ -12,26 +14,51 @@ const finnSamletBruttoForStatus = (andeler, status) => {
   return inntekt;
 };
 
-const finnFrisinnAndelerIPeriodeForStatus = (status, bgPeriode, ytelsegrunnlag) => {
+const erMars2020 = dato => {
+  return dato.month() === 2 && dato.year() === 2020;
+};
+
+const erSøktForMars2020 = frisinnPeriode => {
+  const tom = moment(frisinnPeriode.tom);
+  return erMars2020(tom);
+};
+
+/**
+ Denne metoden sjekker om det er søkt for valgt status i denne beregningsperiode eller i neste dersom den inngår i samme søknadsperiode.
+ Hvis to perioder ligger i samme måned ligger de i samme søknadsperiode
+ */
+const finnFrisinnAndelerISøknadsperiodeForStatus = (status, bgPeriode, ytelsegrunnlag) => {
   const { frisinnPerioder } = ytelsegrunnlag;
   if (!frisinnPerioder) {
     return null;
   }
-  const fom = bgPeriode.beregningsgrunnlagPeriodeFom;
-  const tom = bgPeriode.beregningsgrunnlagPeriodeTom;
-  const matchetPeriode = frisinnPerioder.find(p => p.fom === fom && p.tom === tom);
-  if (!matchetPeriode) {
+  const bgFom = moment(bgPeriode.beregningsgrunnlagPeriodeFom);
+
+  // Søkader for mars må spesialhåndteres da disse er de eneste som krysser måneder
+  if (erMars2020(bgFom) && frisinnPerioder.some(p => erSøktForMars2020(p))) {
```

Comment:
Merk at man vil gå inn her også for første periode (den som starter på skjæringstidspunktet) om denne metoden blir kalt for den perioden.